frontend/src/bundles/auth/components/sign-up-form/styles.module.scss

```diff
@@ -0,0 +1,95 @@
+.form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-top: calc(min(10vh, 228px));
+    margin-bottom: calc(min(10vh, 228px));
+}
+
+.header {
+    width: 100%;
+    max-width: 420px;
+    font-weight: var(--font-weight-bold);
+    font-size: var(--font-size-h4);
+    font-family: var(--font-family);
+    line-height: var(--line-height-main);
+    text-align: center;
+}
+
+.email-sign {
+    margin-top: 27px;
+    margin-bottom: 23px;
+    color: var(--gray-1);
+    font-size: var(--font-size-caption);
+    font-family: var(--font-family);
+}
+
+.input-container {
+    width: 100%;
+
+    &.email {
+        margin-bottom: 25px;
+    }
+
+    &.password {
+        margin-bottom: 8px;
+    }
+}
```

Comment:
why do we have so many duplicate styles?

Reply:
consolidated into form-styles